utils/stripe_utils.py

```diff
@@ -127,22 +127,10 @@ def create_plan(self, amount, name, id):
             id=id)
 
     @handleStripeError
-    def make_payment(self, user, amount, token):
+    def make_payment(self, customer, amount, token):
         charge = self.stripe.Charge.create(
```

Comment:
Luckily, we don't seem to be using this code. This is as silly a mistake as it cam be, imo.